server/src/main/scala/io/delta/sharing/server/CloudFileSigner.scala

```diff
@@ -175,7 +176,12 @@ object AbfsFileSigner {
     val getRelativePathMethod = classOf[AzureBlobFileSystemStore]
       .getDeclaredMethod("getRelativePath", classOf[Path])
     getRelativePathMethod.setAccessible(true)
-    getRelativePathMethod.invoke(abfsStore, path).asInstanceOf[String]
+    var relativePath = getRelativePathMethod.invoke(abfsStore, path).asInstanceOf[String]
+    // remove duplicate separator character for azure relative path
```

Comment:
And could you add this link in the comment as a reference of why we are doing this change: https://github.com/apache/hadoop/commit/af98f32f7dbb9d71915690b66f12c33758011450#diff-94925ffd3b21968d7e6b476f7e85f68f5ea326f186262017fad61a5a6a3815cbL1215

Reply:
done